### PR TITLE
React router playground

### LINKED ISSUE #6: Create a navigation bar

### PR DESCRIPTION
**Pull request by popsps**
fixes #6

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,234 +1,33 @@
-import React, {useState} from "react";
```
Comment (iamamonte) [inline src/App.js:1]: Instead of blowing this away, move the relevant content to a new component. We can call it Playground

File: src/styles/LUX.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/*!
```
Comment (iamamonte) [inline src/styles/LUX.css:1]: Please describe why we need this file.

File: README.md [hunk 3]
```diff
@@ -1,68 +1,98 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (iamamonte) [inline README.md:1]: Roll back these changes. Place your content into a separate readme.

File: package.json [hunk 4]
```diff
@@ -10,6 +10,7 @@
     "react": "^16.13.1",
```
Comment (iamamonte) [inline package.json:10]: Just marking an example of something that should be isolated in its own commit.

File: src/App.css [hunk 5]
```diff
@@ -1,39 +0,0 @@
-.App {
```
Comment (iamamonte) [inline src/App.css:1]: The CSS changes could have been in their own commit with a description along the lines of: "moving css with the app structure"

File: src/components/LoginModal.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
+/**
```
Comment (iamamonte) [inline src/components/LoginModal.js:1]: This file should not be included in this branch because it's not necessary to the functionality we're striving for (adding routing). Since it is already checked in, we won't delete it, but we can remove the reference to it in navigation.js

File: src/components/Navigation.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+          <Link className="nav-link" to='/'>Home</Link>
+          <Link className="nav-link" to='/events'>Events</Link>
+          <Link className="nav-link" to='/conversations'>Conversations</Link>
+          <NavDropdown title="Interns" id="basic-nav-dropdown">
+            <Link className="dropdown-item" to='/interns/shayan'>Shayan</Link>
+          </NavDropdown>
+        </Nav>
+        <Button variant="outline-success" onClick={() => setModalVisibility(true)}>Login</Button>
+      </Navbar.Collapse>
+      <LoginModal handleClose={() => setModalVisibility(false)} show={modalVisibility}/>
```
Comment (iamamonte) [inline src/components/Navigation.js:27]: Per previous comment, remove this as it is not necessary to the functionality we're going for (also the import startement)

**Comment by iamamonte** [comment]
Please resolve the warnings as well. They are causing build failures in the github checks

File: src/styles/LUX.css (hunk 2 above)
Comment (popsps) [reply src/styles/LUX.css:1]: This one is responsible for styling of Navbar. We can remove and replace it back with bootstrap.min.css.

**Comment by iamamonte** [comment]
In the future, we want to break apart commits like this one: https://github.com/Turnout-Activism/ReactPlayground/pull/24/commits/c8e1420882d86d5d4ebacda86ee3e36a08e63640

... into separate commits like so:

-resolve warnings
-remove lux
-remove login modal
-readme changes

File: README.REACT-ROUTER.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+cd my-react-app-with-router
+npm i react-router-dom
+```
+
+Additionally if you want typescript or better intellisense you can install `@types/react-router` as dev dependencies.
+
+```sh
+npm i -D @types/react-router
+```
+
```
Comment (iamamonte) [inline README.REACT-ROUTER.md:22]: Because we've already added it as a dependency to this repo, people can just run npm install to pull it down. I think this section is accurate from the perspective of starting from scratch, but we may want to specific that people don't need to do that for this repo.

File: README.REACT-ROUTER.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+For this project React router is already set up for you. For this assignment `App.js` serves as our main page. Any additional component/page needs to be registered at `App.js`. Look at how we structured `App.js`. Here is what you need to do.
+
+- Create a Component in `components/interns` and name it `internNamePage`.
+- Customize your component/page however you want (add personal information etc).
+- If you want to add custom styling, you can add it in `styles/app.css`.
+- Register the path `/interns/internName` with your component in `Route`.
+- Add a Link to your component/page in the `Navigation.js` under the `NavDropdown`.
+
+That is it.
```
Comment (iamamonte) [inline README.REACT-ROUTER.md:98]: Let's remove this and place it elsewhere. Perhaps we can modify the ShayanPage to be React Task 2 (Routing) and put the instructions there.

File: package.json [hunk 10]
```diff
@@ -35,6 +36,7 @@
   },
   "devDependencies": {
     "@types/jest": "^26.0.4",
+    "@types/react-router-dom": "^5.1.5",
```
Comment (iamamonte) [inline package.json:39]: Did we need this dev dependency?
